### DISCUSSION TITLE
Configuration changes in Hyper.is Terminal [Canary]

### DISCUSSION THREAD

**Discussion by clarityam** [cmderdev/cmder#2936] [General]
Vercel has moved away from their previous configuration file of .hyper.js to hyper.json in newer builds of Hyper Terminal which changes the syntax of using CMDER with Hyper. 

## Previous configuration:

```
    shell: 'cmd.exe',
    shellArgs: ['/k', '[cmder_root]\\vendor\\init.bat'],
    env: { 'TERM':'cygwin' }, // required for git-for-windows
```

# New configuration:

```
    "shell": "cmd.exe",
    "shellArgs": [
      "--login", "/k", "Title CMDER & %CMDER_ROOT%\\vendor\\init.bat"
    ],
    "env": {"TERM":"cygwin"},
```

In total: Moving to double quotes. It is a simple but easy fix. if you were having issues with Canary, I hope this helps!

Reference to Prior Config: [Seamless Hyper integration](https://github.com/cmderdev/cmder/wiki/Seamless-Hyper-integration)

**Comment by DRSDavidSoft** [comment]
Hi there @clarityam, thanks for the notice. Can you please also modify the Wiki using this link to reflect changes:
https://github.com/cmderdev/cmder/wiki/Seamless-Hyper-integration/_edit

Appreciate it!

**Reply by clarityam** [reply]
Sure, I can do that! 
One issue, however, is when trying your link, it returns "You do not have permission to update this wiki. "

**Reply by DRSDavidSoft** [reply]
@MartiUK Can you please open editing the Wiki to the public? I assume it was disabled due to the spam, but I believe it'd be safe to do so now.

@clarityam Thanks for the notes, I'd like to ask again for editing the Wiki once the permissions to do so are granted. Once again thanks for the contribution! 👍🏻

**Reply by MartiUK** [reply]
Done, although I will close it again if there's any more spam.

**Reply by clarityam** [reply]
Added! Feel free to revise any mistakes I made.

**Reply by DRSDavidSoft** [reply]
@MartiUK Duly noted, thank you for it!

@clarityam Excellent! If I may ask for one more favor, could you please attach a high-quality screenshot of your Hyper Canary + Cmder setup that we could use in the Wiki?

**Reply by clarityam** [reply]
Sure! I have a few different versions here, the normal look, the one stated in the guide, and those two plus the versions of Hyper.
![Hyper_igJQh9boPH](https://github.com/cmderdev/cmder/assets/40682937/abd453fc-53e7-4789-847c-ef305fa4b6e8)
![Hyper_oSRQ48e0r3](https://github.com/cmderdev/cmder/assets/40682937/227a0c0a-ea56-43de-b887-be29b29baa46)
![Hyper_cHeLdSDFa9](https://github.com/cmderdev/cmder/assets/40682937/301ffbb9-edd5-4a7e-89e0-fecbebc4a53c)
![Hyper_5dimw9ID3n](https://github.com/cmderdev/cmder/assets/40682937/f4f0fa76-e6ed-4bec-bdbc-05c25adec305)

Here is also an Imgur link with the same images: https://imgur.com/a/IhbWtPK
